packages/client/src/App.test.tsx

```diff
@@ -1,9 +1,75 @@
-import React from "react"
-import ReactDOM from "react-dom"
+import { ListItem } from "@material-ui/core"
+import { createHistory, createMemorySource } from "@reach/router"
+import { ReactWrapper } from "enzyme"
+import * as React from "react"
 import App from "./App"
+import Avatar from "./Avatar"
+import { mount, updates } from "./testing"
+import * as $ from "./testing/fixtures"
 
-it("renders without crashing", () => {
-  const div = document.createElement("div")
-  ReactDOM.render(<App />, div)
-  ReactDOM.unmountComponentAtNode(div)
+it("displays a list of conversations", async () => {
+  const app = mount(<App />, {
+    mocks: [$.getAccountMock],
+    route: "accounts/1/dashboard"
+  })
```

Comment:
The `mount` test helper renders a React component for testing. It sets up a provider to handle GraphQL queries originating from the component. It also sets up a `LocationProvider` to provide the initial URL set up by the test, and to keep track of any URL changes that components make.

In most cases you will need to provide an array of mocks. Each of these is a predefined response to a GraphQL query. You will need to provide one mock for each query. If the same query runs more than once, you will need to specify a mock for each request.